src/views/WellnessCheck/index.js

```diff
@@ -0,0 +1,215 @@
+import Grid from '@material-ui/core/Grid';
+import React, { Component } from 'react';
+import user from '../../services/user';
+import Login from '../Login';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardHeader from '@material-ui/core/CardHeader';
+import Approved from './components/Approved/Approved';
+import Denied from './components/Denied/Denied';
+import wellness from '../../services/wellness';
+import GordonLoader from '../../components/Loader';
+import '../../app.css';
+import { gordonColors } from './../../theme';
+
+export default class WellnessCheck extends Component {
+  constructor(props) {
+    super(props);
+
+    this.logIn = this.logIn.bind(this);
+
+    this.state = {
+      personType: null,
+      network: 'online',
+      currentStatus: 'I am symptomatic',
```

Comment:
it would seem like the wording is being used but it actually is not haha, its just a variable that the ui uses to determine certain things. Could of been true or false but I thought it would make the code easier to follow if the new exactly what we were checking for at all times.

Reply:
I would suggest declaring constants for this.  Perhaps
   const SYMPTOMS = true;
   const NO_SYMPTOMS = false;
This is clear, and I think will allow autocomplete to help with spelling.

Reply:
fixed variables so that now are boolean.  
SYMPTOMS = true;
NO_SYMPTOMS = false;